lib/led/ledcontrol.js

```diff
@@ -80,18 +79,7 @@ function LedControl(opts) {
     controller = Controllers.DEFAULT;
   }
 
-  // functions from Controller interface
-
-  this.clear = controller.clear;
-  this.led = controller.led;
-  this.row = controller.row;
-  this.scanLimit = controller.scanLimit;
-  this.send = controller.send;
-  this.sendDigit = controller.sendDigit;
-  this.initialize = controller.initialize;
-
-  // controller specific op codes
-  this.OP = controller.OP;
+  Board.Controller.call(this, controller, opts);
```

Comment:
Yep, that's [the order the motor module takes](https://github.com/rwaldron/johnny-five/blob/master/lib/motor.js#L529-L544), I'll update accordingly.